python/baseline/tf/seq2seq/model.py

```diff
@@ -466,7 +466,7 @@ def predict(self, batch_dict):
         if len(vec.shape) == 3:
             return vec.transpose(1, 2, 0)
         else:
-            return vec.transpose(1, 0)
+            return [vec.transpose(1, 0)]
```

Comment:
Lets not change anything from this class. Its getting refactored.   Assume for now a 3D tensor, as that will be the case after my latest mods